pkg/specgen/generate/storage.go

```diff
@@ -20,9 +20,10 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-var (
-	errDuplicateDest = errors.Errorf("duplicate mount destination")
-)
+var errDuplicateDest = errors.Errorf("duplicate mount destination")
+
+// The default path of the mounted container init.
+const containerInitPath = "/run/podman-init"
```

Comment:
Can we move this into `libpod/define` and use it in libpod/diff.go as well.

Reply:
Done :heavy_check_mark: